scenarioo-client/app/manage/builds/build-list.component.html

```diff
@@ -0,0 +1,160 @@
+                    </span>
+                </th>
+
+                <th (click)="setOrder('buildDescription.date')" [class.selected]="order === 'buildDescription.date'">
+                    Date
+                    <span *ngIf="order !== 'buildDescription.date'">
+                        <i class="fas fa-sort"></i>
+                    </span>
+                    <span *ngIf="order === 'buildDescription.date' && reverse">
+                        <i class="fas fa-sort-down"></i>
+                    </span>
+                    <span *ngIf="order === 'buildDescription.date' && !reverse">
+                        <i class="fas fa-sort-up"></i>
+                    </span>
+                </th>
+
+                <th (click)="setOrder('buildDescription.revision')" [class.selected]="order === 'buildDescription.revision'">
+                    Revision
+                    <span *ngIf="order !== 'buildDescription.revision'">
+                        <i class="fas fa-sort"></i>
+                    </span>
+                    <span *ngIf="order === 'buildDescription.revision' && reverse">
+                        <i class="fas fa-sort-down"></i>
+                    </span>
+                    <span *ngIf="order === 'buildDescription.revision' && !reverse">
+                        <i class="fas fa-sort-up"></i>
+                    </span>
+                </th>
+
+                <th>Use Cases</th>
+                <th>Scenarios</th>
+
+                <th (click)="setOrder('buildDescription.status')" [class.selected]="order === 'buildDescription.status'">
+                    Build Status
+                    <span *ngIf="order !== 'buildDescription.status'">
+                        <i class="fas fa-sort"></i>
+                    </span>
+                    <span *ngIf="order === 'buildDescription.status' && reverse">
+                        <i class="fas fa-sort-down"></i>
+                    </span>
+                    <span *ngIf="order === 'buildDescription.status' && !reverse">
+                        <i class="fas fa-sort-up"></i>
+                    </span>
+                </th>
+
+                <th (click)="setOrder('status')" [class.selected]="order === 'status'">
+                    Import Status
+                    <span *ngIf="order !== 'status'">
+                        <i class="fas fa-sort"></i>
+                    </span>
+                    <span *ngIf="order === 'status' && reverse">
+                        <i class="fas fa-sort-down"></i>
+                    </span>
+                    <span *ngIf="order === 'status' && !reverse">
+                        <i class="fas fa-sort-up"></i>
+                    </span>
+                </th>
+
+                <th>Details & Log</th>
+                <th>Reimport</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr *ngFor="let build of buildImportStates | orderBy: order:reverse:true | scSearchFilter: searchTerm">
```

Comment:
use the async pipe then you can subscribe to an observable

Reply:
Nice one! Thx for the hint.
Done